R/create_data_table_templates.R

```diff
@@ -0,0 +1,105 @@
+                                        Mass = NA_real_,
+                                        Tarsus = NA_real_,
+                                        OriginalTarsusMethod = NA_character_,
+                                        WingLength = NA_real_,
+                                        Age_observed = NA_integer_,
+                                        Age_calculated = NA_integer_,
+                                        ChickAge = NA_integer_,
+                                        ExperimentID = NA_character_)
+
+## Save
+save(capture_data_template, file = "./data/Capture_data_template.rda")
+
+########################################################################
+########################################################################
+
+## Individual data template
+individual_data_template <- tibble::tibble(IndvID = NA_character_, ## Missing data not allowed
+                                           Species = NA_character_, ## Missing data not allowed
+                                           PopID = NA_character_, ## Missing data not allowed
+                                           BroodIDLaid = NA_character_,
+                                           BroodIDFledged = NA_character_,
+                                           RingSeason = NA_integer_,
+                                           RingAge = NA_character_,
+                                           Sex_calculated = NA_character_,
+                                           Sex_genetic = NA_character_)
+
+## Save
+save(individual_data_template, file = "./data/Individual_data_template.rda")
+
+########################################################################
+########################################################################
+
+## Brood data template
+brood_data_template <- tibble::tibble(BroodID = NA_character_, ## Missing data not allowed
+                                      PopID = NA_character_, ## Missing data not allowed
+                                      BreedingSeason = NA_integer_, ## Missing data not allowed
+                                      Species = NA_character_, ## Missing data not allowed
+                                      Plot = NA_character_,
+                                      LocationID = NA_character_,
+                                      FemaleID = NA_character_,
+                                      MaleID = NA_character_,
+                                      ClutchType_observed = NA_character_,
+                                      ClutchType_calculated = NA_character_,
+                                      LayDate_observed = as.Date(NA),
+                                      LayDate_min = as.Date(NA),
+                                      LayDate_max = as.Date(NA),
+                                      ClutchSize_observed = NA_integer_,
+                                      ClutchSize_min = NA_integer_,
+                                      ClutchSize_max = NA_integer_,
```

Comment:
True. Do you think it would be better to change the column class here (even though it would then differ from the standard protocol) or keep it as NA_integer_?